core/templates/pages/contributor-dashboard-page/contributor-badge/contributor-badge.component.html

```diff
@@ -0,0 +1,552 @@
+                            </translation-badge>
+                        </div>
+                    </div>
+                </mat-card>
+            </div>
+        </div>
+        <div [hidden]="showMobileTranslationBadges">
+            <div [hidden]="dataLoading || (!userCanSuggestQuestions && !userCanReviewQuestionSuggestions)" class="badge-type-text">Questions</div>
+            <div *ngIf="!dataLoading && userHasQuestionRights">
+                <mat-card layout="row" class="list-view-item oppia-contributions-required-message-container">
+                    <div *ngIf="userCanSuggestQuestions" class="badge-box">
+                        <div>Submission</div>
+                        <div class="badge-container">
+                            <question-badge class="badge-item" *ngFor="let badge of questionSubmissionBadges" isUnlocked="{{badge.isUnlocked}}" type="{{badge.text}}" value="{{badge.contributionCount}}">
+                            </question-badge>
+                        </div>
+                    </div>
+                    <div *ngIf="userCanReviewQuestionSuggestions" class="badge-box">
+                        <div>Review</div>
+                        <div class="badge-container">
+                            <question-badge class="badge-item" *ngFor="let badge of questionReviewBadges" isUnlocked="{{badge.isUnlocked}}" type="{{badge.text}}" value="{{badge.contributionCount}}">
+                            </question-badge>
+                        </div>
+                    </div>
+                    <div *ngIf="userCanReviewQuestionSuggestions" class="badge-box">
+                        <div>Correction</div>
+                        <div class="badge-container">
+                            <question-badge class="badge-item" *ngFor="let badge of questionCorrectionBadges" isUnlocked="{{badge.isUnlocked}}" type="{{badge.text}}" value="{{badge.contributionCount}}">
+                            </question-badge>
+                        </div>
+                    </div>
+                </mat-card>
+            </div>
+        </div>
+    </div>
+    <mat-card [hidden]="dataLoading" *ngIf="(showMobileTranslationBadges && !translationBadges[selectedLanguage])" layout="row" class="contributions-required-container mobile-badge-container list-view-item">
+      <div class="oppia-contributions-required-message">
+        Start contributing to see your badges!
+      </div>
+    </mat-card>
+    <mat-card [hidden]="dataLoading" *ngIf="!translationBadges[selectedLanguage] && !userHasQuestionRights" layout="row" class="contributions-required-container desktop-badge-container list-view-item">
+      <div class="oppia-contributions-required-message">
+        Start contributing to see your badges!
+      </div>
+    </mat-card>
```

Comment:
Can we add tests for these two cases? Could you also post screenshots of these two cases as well?

Reply:
When has no translation badges AND no question rights(no question badges).

![image](https://user-images.githubusercontent.com/41409951/202260337-c01e3902-44f7-4744-9a07-f37055a2c07f.png)

When has no translation badges AND has question rights(has unlocked question badges)
![image](https://user-images.githubusercontent.com/41409951/202260737-e8e41ccd-995f-4ab3-a05b-300b1ba80972.png)

When has translation badges AND no question rights(no question badges)
![image](https://user-images.githubusercontent.com/41409951/202261442-359df1f2-9a55-4ec6-a3ec-94879d02fdc5.png)

Updated test cases